models/stored/batched_tx.go

```diff
@@ -2,38 +2,42 @@ package stored
 
 import (
 	"bytes"
+	"fmt"
 
 	"github.com/Worldcoin/hubble-commander/models"
+	"github.com/pkg/errors"
+	bh "github.com/timshannon/badgerhold/v4"
 )
 
 var (
-	BatchedTxName   = models.GetTypeName(BatchedTx{})
-	BatchedTxPrefix = models.GetBadgerHoldPrefix(BatchedTx{})
+	BatchedTxName                = models.GetTypeName(BatchedTx{})
+	BatchedTxPrefix              = models.GetBadgerHoldPrefix(BatchedTx{})
+	errInvalidBatchedTxIndexType = fmt.Errorf("invalid models.BatchedTx index type")
 )
 
 type BatchedTx struct {
 	PendingTx
-	CommitmentID models.CommitmentID `badgerhold:"index"`
+	CommitmentSlot models.CommitmentSlot
```

Comment:
If you insist then I can make this change! However, if I can push back a little bit, this change will make the codebase harder to grep, I tend to prefer more unique field names such as `CommitmentSlot`, which makes it easier to develop using simpler tooling like grep and vim.

Reply:
As a result of naming this field `CommitmentSlot` some methods end up having wierd-sounding names and params: 
- `getTransactionSlotsByBatchID` could be `getTransactionIDsByBatchID`
- `MarkTransactionsAsPending(txSlots)` instead of `MarkTransactionsAsPending(txIDs)`
  
For me `tx.ID` communicates better that the field is used as a primary key than`tx.CommitmentSlot`. You also know that it is a `CommitmentSlot` just looking at the type, so field name doesn't introduce any new information now.

Anyway I don't insist, feel free to leave it as it is if you're not convinced.

Reply:
Just made the change!